### DISCUSSION TITLE
F-Droid versus others [Help Wanted]

### DISCUSSION THREAD

**Discussion by hhmacedo** [standardnotes/forum#3098] [General]
Hello, there 👋

I'm trying to understand what's the difference between using F-Droid (my version) and others sources (like apk and Play Store). 

The version available on F-Droid it's the 3.160.16:
![Screenshot_20230530_233450_F-Droid](https://github.com/standardnotes/forum/assets/22848633/b9ef71cf-f631-4b2b-b11b-ce84a1f9ae74)

But when I open the What's New section it shows two more versions, including one that fix the bug that I'm facing right now.
![Screenshot_20230530_233523_Standard Notes](https://github.com/standardnotes/forum/assets/22848633/bbe59a83-4012-4e41-979f-0e5735393138)

Why the lag on F-Droid, should I avoid using it?

**Comment by effieeee** [comment]
Hi, the difference would be that the team only officially maintains the APK on the Play Store, and of course, on our official repo. Other packages, such as the one on F-droid, are not officially maintained by anyone from Standard Notes, so there are usually delays with updates on those platforms 🙏
